### PR TITLE
Rework keygen

### PR DESCRIPTION
**Pull request by NaNuNaNu**
Rework keygen according to issue: https://github.com/0chain/keygen/issues/4

### CODE REVIEW

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by NaNuNaNu** [comment]
@dabasov really not sure about the docker setup this will run with so would be good to have your input here.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module github.com/0chain/keygen/code/go/0chain.net
```
Comment (peterlimg) [inline go.mod:1]: Please update the module to `github.com/0chain/keygen`

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**